jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -691,7 +700,18 @@ private void addFileToLayer(
     layerBuilders.get(layerType).addEntry(sourceFile, pathInContainer, modificationTime);
   }
 
-  private void addDirectoryContentsToLayer(
+  /**
+   * Adds the contents of a directory to a {@link FileEntriesLayer.Builder}.
+   *
+   * @param layerBuilders map that keeps track of all layers, with the key being the layer type and
+   *     value being a {@link FileEntriesLayer.Builder}.
+   * @param layerType the layer type.
+   * @param sourceRoot path to root directory.
+   * @param pathFilter predicate to filter the files by.
+   * @param basePathInContainer base path in container.
+   * @throws IOException if I/O error occurs when opening the directory.
+   */
+  public static void addDirectoryContentsToLayer(
```

Comment:
Should remain an instance method.

And note `JavaContainerBuilder` is a public API. Probably we don't want to expose this internal helper method.

Just asking: did we decide to use `JavaContainerBuilder` instead of `JibContainerBuilder` for some reason? It's fine if `JavaContainerBuilder` works, but just saying I believe it's a bit curated and limiting.

Reply:
Thanks for pointing this out! No longer using JavaContainerBuilder in JarProcessor.